### PR TITLE
ChromeDriverProvider

### PR DESCRIPTION
**Pull request by serhiibastrykin**

### CODE REVIEW

File: src/main/java/com/seleniumeasy/alertsandmodals/ProgressBarModalPage.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public ProgressBarModalPage openDialogWithCustomSettings() {
+        buttonDialogWithCustomSettings.click();
+        return this;
+    }
+
+    public void verifySimpleDialogIsDisplayed(Condition c) {
+        modalDialog.should(c);
+    }
+
+    public void verifyDialogWithCustomMessageIsDisplayed(Condition c) {
```
Comment (ivan-harasym) [inline src/main/java/com/seleniumeasy/alertsandmodals/ProgressBarModalPage.java:34]: wrong name of PA method: you can put Condition.enabled or Condition.hidden and in that case you will have inconsistency
Please update it for all methods where you pass Condition as a parameter
you can name method like 'verifyDialogWithCustomMessage'

File: src/main/java/com/seleniumeasy/listbox/DataListFilterPage.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class DataListFilterPage extends DemoHomePage {
+    private final SelenideElement inputSearchAttendees = $("#input-search");
+    private final ElementsCollection companies = $$(".info-block h5"),
+            names = $$(".info-block h4"),
+            titles = $$(".info-block p"),
+            phones = $$x("//span[contains(text(), 'Phone')]"),
+            emails = $$x("//span[contains(text(), 'Email')]");
+
+    public DataListFilterPage enterSearchText(String searchText) {
+        inputSearchAttendees.sendKeys(searchText);
```
Comment (ivan-harasym) [inline src/main/java/com/seleniumeasy/listbox/DataListFilterPage.java:18]: please do not use Selenium methods
sendKeys is not method of Selenide

File: src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    public JQueryListBoxPage addItemsToResultList(String[] names) {
+        for (String name : names) {
+            pickList.find(exactText(name)).click();
+        }
+        buttonAdd.click();
+        return this;
+    }
+
+    public JQueryListBoxPage removeItemFromResultList(String name) {
```
Comment (ivan-harasym) [inline src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java:35]: method duplication: 
please merge removeItemFromResultList(String name) and removeItemsFromResultList(String[] names)
you can use VarArgs

File: src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    }
+
+    public JQueryListBoxPage validateItemIsNotDisplayedInResultList(String name) {
+        resultList.find(exactText(name)).shouldNot(exist);
+        return this;
+    }
+
+    public void validateItemsAreDisplayedInPickList(String[] names) {
+        for (String name : names) {
+            pickList.find(exactText(name)).should(exist);
```
Comment (ivan-harasym) [inline src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java:90]: Condition.exist will not check if element is visible.
To make sure element is visible in list please use Condition.visible

Comment (serhiibastrykin) [reply src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java]: Replace condition 'exist' by 'visible'

File: src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java (hunk 3 above)
Comment (serhiibastrykin) [reply src/main/java/com/seleniumeasy/listbox/JQueryListBoxPage.java]: Delete all duplicate methods in JQueryListBoxPage class. Replace parameters by VarArgs. Also, modify tests

File: src/main/java/com/seleniumeasy/listbox/DataListFilterPage.java (hunk 2 above)
Comment (serhiibastrykin) [reply src/main/java/com/seleniumeasy/listbox/DataListFilterPage.java]: Replace sendKeys() by setValue()

File: src/main/java/com/seleniumeasy/alertsandmodals/ProgressBarModalPage.java (hunk 1 above)
Comment (serhiibastrykin) [reply src/main/java/com/seleniumeasy/alertsandmodals/ProgressBarModalPage.java]: Rename all methods of the class in a style like 'verifyDialogWithCustomMessage'

**Pull request merged**